### PR TITLE
Add tests for formik select

### LINKED ISSUE #1070: Add tests for Formik Select

### PR DESCRIPTION
**Pull request by mohit-s96**
closes #1070

### CODE REVIEW

**Comment by mohit-s96** [comment]
@ajmaln _a please review

**Comment by ajmaln** [comment]
@mohit-s96 _a Please add tests to verify error messages are showing up when added validaitonSchema.

**Comment by mohit-s96** [comment]
@ajmaln _a please review. I have used a custom validator function because using `validationSchema` kept throwing errors. The errors returned by formik hook [here](https://github.com/bigbinary/neeto-ui/blob/696d914b5e9481ba307e7735f7ff31b301099316/lib/components/formik/Select.js#L44) when using a nested `yup` schema are objects and thus can't be rendered directly.

File: tests/formik/Select.test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      expect(onSubmit).toHaveBeenCalledWith({
+        formikSelect: SELECT_OPTIONS[0],
+      })
+    );
+  });
+
+  it("should display validation error when invalid input is provided", async () => {
+    render(<SelectTest onSubmit={() => {}} />);
+    userEvent.click(screen.getByText("Submit"));
+    screen.debug();
```
Comment (ajmaln) [inline tests/formik/Select.test.js:124]: remove `screen.debug()`

**Review by ajmaln — COMMENTED** [review]
@mohit-s96 _a Please check the minor change

**Comment by mohit-s96** [comment]
@ajmaln _a please review

**Pull request merged**